### ISSUE TITLE
Aborting program due to exceptional state: locale::facet::_S_create_c_locale name not valid

### ISSUE THREAD

**Issue by bugduino** [Creepsky/creepMiner#122]
Hi, I've the following error when starting creepMiner on linux

```
user@vmi:~/creepMiner/bin$ ./creepMiner 
13:45:35: creepMiner 1.6.0 Unix x64
13:45:35: ----------------------------------------------
13:45:35: Github:   https://github.com/Creepsky/creepMiner
13:45:35: Author:   Creepsky [creepsky@gmail.com]
13:45:35: Burst :   BURST-JBKL-ZUAV-UXMB-2G795
13:45:35: ----------------------------------------------
13:45:35: Based on http://github.com/uraymeiviar/burst-miner
13:45:35: author : uray meiviar [ uraymeiviar@gmail.com ]
13:45:35: please donate to support developments :
13:45:35:  [ Burst   ] BURST-8E8K-WQ2F-ZDZ5-FQWHX
13:45:35:  [ Bitcoin ] 1UrayjqRjSJjuouhJnkczy5AuMqJGRK4b
13:45:35: ----------------------------------------------
13:45:35: using config file mining.conf
13:45:35: Total plots size: 923.23 GB
13:45:35: Submission Max Retry : 3
13:45:35: Buffer Size : 128 MB
13:45:35: Pool Host : http://pool.burstcoin.eu:80 (216.165.179.116)
13:45:35: Mininginfo URL : http://pool.burstcoin.eu:80 (216.165.179.116)
13:45:35: Wallet URL : https://wallet.burst-team.us:8128 (216.165.179.44)
13:45:35: Server URL : http://192.168.0.100:8080 (0.0.0.0)
13:45:35: Aborting program due to exceptional state: locale::facet::_S_create_c_locale name not valid
```

any ideas?

**Comment by bugduino** [comment]
Ok I just found the solution here https://stackoverflow.com/questions/19100708/mongodb-mongorestore-failure-localefacet-s-create-c-locale-name-not-valid it turns out that I need to do the following `export LC_ALL="en_US.UTF-8"`

**Comment by theblazehen** [comment]
@bugduino Did you by any chance upgrade from ubuntu 16.04 to 17.04? That broke it for me as well

**Comment by blub0hr** [comment]
I got the same exception when running `run.sh` which had `export LC_ALL="en_US.UTF-8"`. Removing that line solved it for me on an odroid xu4 with ubuntu 16.04.

**Comment by llybin** [comment]
Archlinux, issue is reproduce.
Solve problem: export LC_ALL=C.UTF-8 -> export LC_ALL=en_US.UTF-8 in run.sh
